hmt_escrow/storage.py

```diff
@@ -10,14 +10,33 @@
 from p2p import ecies
 from ipfshttpclient import Client
 
+import boto3
+from botocore.client import Config
+
 SHARED_MAC_DATA = os.getenv(
     "SHARED_MAC",
     b'9da0d3721774843193737244a0f3355191f66ff7321e83eae83f7f746eb34350')
 
+logging.getLogger("boto").setLevel(logging.INFO)
+logging.getLogger("botocore").setLevel(logging.INFO)
+logging.getLogger("boto3").setLevel(logging.INFO)
+
 LOG = logging.getLogger("hmt_escrow.storage")
+DEBUG = "true" in os.getenv("DEBUG", "false").lower()
+LOG.setLevel(logging.DEBUG if DEBUG else logging.INFO)
+LOCAL = os.getenv("LOCAL", False)
```

Comment:
might wanna comment on what this means .. I see the comment below but its a bit nebulous up here.

Reply:
removed the whole variable since it's not needed anymore